tracpro/polls/views.py

```diff
@@ -191,6 +191,31 @@ def get_form_kwargs(self):
 
         def form_valid(self, form):
             form.save()
+
+            # Save the associated Questions for this poll here
+            # now that these polls have been activated for the Org
+            selected_polls = []
+            for poll in form.cleaned_data['polls']:
+                selected_polls.append(poll.name)
+
+            org = self.request.org
+            temba_polls = org.get_temba_client().get_flows(archived=False)
```

Comment:
The rest of this looks very much like PollManager.sync(), except that this only operates on the org's selected polls instead of all of them. Maybe there's some opportunity here to reduce duplication of code?